Gemfile

```diff
@@ -1,16 +1,26 @@
 source 'https://rubygems.org'
 
-gemspec
-
 group :development do
-  gem 'vagrant',      git: 'https://github.com/mitchellh/vagrant.git'
-  gem 'rake'
+  gem 'vagrant',
+      :git => 'git://github.com/mitchellh/vagrant.git',
+      :ref => 'v1.8.1'
+
+  # test dependencies
+  gem 'minitest'
+  gem 'mocha'
+
   gem 'cucumber', '~> 2.1'
   gem 'aruba', '~> 0.13'
   gem 'komenda', '~> 0.1.6'
   gem 'launchy'
   gem 'gem-compare'
   gem 'mechanize'
+
+  # build tool
+  gem 'rake', '10.4.2'
```

Comment:
> Why not lock listen instead, if listen is the problem?

That is not the big issue here. The big issue is that the dev branch of Vagrant switched to Ruby 2.2 as well. So even if you try to lock the listen version, you still need to deal with the change of Ruby runtime.